Sources/NIOHTTP1/HTTPDecoder.swift

```diff
@@ -187,6 +195,14 @@ public final class HTTPResponseDecoder: HTTPDecoder<HTTPClientResponsePart>, Cha
     }
 }
 
+/// Strategy to use when a decoder is removed from the pipeline and an HTTP upgrade was detected before.
```

Comment:
Nit: let's write this as "Strategy to use when a `HTTPDecoder` is removed from a pipeline after a HTTP upgrade was detected".

Reply:
addressed... will merge once green if I not hear back :)